src/controller/tests/TestWriteChunking.cpp

```diff
@@ -454,13 +478,149 @@ void TestWriteChunking::TestNonConflictWrite(nlTestSuite * apSuite, void * apCon
     emberAfClearDynamicEndpoint(0);
 }
 
+namespace TestTransactionalListInstructions {
+
+using PathStatus = std::pair<app::ConcreteAttributePath, CHIP_ERROR>;
+struct Instructions
+{
+    // The paths used in write request
+    std::vector<ConcreteAttributePath> paths;
+    // operations on OnListWriteBegin and OnListWriteEnd on the server side
+    std::function<void(std::unique_ptr<WriteClient> & writeClient, const app::ConcreteAttributePath & path)> onListWriteBegin;
```

Comment:
Would be good to rename this to `onListWriteBeginActions`.

Reply:
Done